### PR TITLE
Admin Generator (Future): Delete generated files prior to generating

### PR DESCRIPTION
**Pull request by Ben-Ho**
## Requirement
Fix/improve regenerate a single config-file for easier development. Previously there where (lint-)issues if the generated file already existed.

## Solution
Remove the generated file before writing.

## Open issues
 - [x] ~~If changed config wouldn't create the same files, previously created wont be deleted~~ We can neglect this

### CODE REVIEW

**Comment by nsams** [comment]
why not delete always?

**Review by nsams — DISMISSED** [review]

File: packages/admin/cms-admin/src/generator/future/generator.ts [hunk 1]
```diff
@@ -119,6 +120,11 @@ export async function runFutureGenerate(filePattern = "src/**/*.cometGen.ts") {
         const configs = await import(`${process.cwd()}/${file.replace(/\.ts$/, "")}`);
         //const configs = await import(`${process.cwd()}/${file}`);
 
+        const codeOuputFilename = `${targetDirectory}/${basename(file.replace(/\.cometGen\.ts$/, ""))}.tsx`;
+        await fs.rm(codeOuputFilename).catch((e) => false);
```
Comment (johnnyomair) [inline packages/admin/cms-admin/src/generator/future/generator.ts:124]: Should we really suppress errors like this?

**Review by johnnyomair — COMMENTED** [review]
We can probably remove the rimraf call in the future: https://github.com/vivid-planet/comet/blob/main/demo/admin/package.json#L6

**Comment by johnnyomair** [comment]
>  If changed config wouldn't create the same files, previously created wont be deleted

This should be covered by `rimraf 'src/*/generated'` here: https://github.com/vivid-planet/comet/blob/main/demo/admin/package.json#L6.

**Comment by Ben-Ho** [comment]
> > If changed config wouldn't create the same files, previously created wont be deleted
> 
> This should be covered by `rimraf 'src/*/generated'` here: https://github.com/vivid-planet/comet/blob/main/demo/admin/package.json#L6.

you're right. but this does only work if you want to regenerate all configs, not for single ones so I mentioned it. This change was mainly to better support regenerate a single config.

**Review by johnnyomair — APPROVED** [review]

**Pull request merged**